updater/README.md

```diff
@@ -11,32 +11,35 @@ GitHub network, and so is not generally accessible.
 
 ## Setup
 
-You will need to provide the build a Personal Access Token to access the GitHub Package Registry to retrieve
-dependency containers.
+To work with the updater, you will need to build the bundler image using the
+build script in this directory:
 
-[Create a token](https://github.com/settings/tokens/new) with the `packages:read` scope and set it in your environment
-as `GPR_TOKEN`
-
-Run the setup script:
-
-```
-script/setup
+```bash
+script/build
 ```
 
 ## Tests
 
 We run [rspec](https://rspec.info/) tests inside a Docker container for this project:
 
-```
+```bash
 script/test
 ```
 
 You can run an individual test file like so:
 
-```
+```bash
 script/test spec/dependabot/integration_spec.rb
```

Comment:
I'm not sure I'm in love with going this route, since for failures the `rspec` output spits out
```
rspec <filepath>:<line>
```

so for our drive-by contributors who are unfamiliar with `rspec` or even ruby in general (that was me a year ago!) seeing this in the docs leaves me confused on how do I run that particular test because I'm supposed to run `script/test`, not `rspec`... resulting in "??!" followed by half an hour poking through scripts to see how things are wired up.

I've also found it a little slow when each test run involves spinning up `docker run`...

For all the ecosystem-specific tests, we've gone the route of recommending dropping into the `docker-dev-shell`, then `cd <folder> && rspec` because then it's easy to see how to pass flags directly to `rspec`. Ideally we could continue that pattern for `updater`, even though it's not technically an ecosystem, as that's what our drive-by contributors will be used to.

Is there a reason we can't go that route? The requesite folders are already mounted in the `docker-dev-shell`:
* https://github.com/dependabot/dependabot-core/pull/6569

Reply:
You got a point @jeffwidman, I like the idea of keeping the `docker-dev-shell` pattern.

Reply:
That's a fair point, I think I approached this as a 'fix what is broken' based on my muscle memory of working on updater when it lived separately.

I'll try that workflow today and see how I get on and if there are any caveats for the updater.